demo/docbot/cluster_bootstrap.py

```diff
@@ -165,21 +167,22 @@ def initialize_connector(client: MLClient) -> None:
         print("Connector 'Cohere Connector' already exists. Skipping initialization.")
 
 
-def initialize_model(client: MLClient) -> None:
+def initialize_model(client: MLClient, model_data=None) -> None:
```

Comment:
We can remove the model_data here and can instead just pull "name" out and have that be variable.